pallets/portfolio/src/benchmarking.rs

```diff
@@ -22,18 +22,17 @@ use polymesh_common_utilities::{
 use polymesh_primitives::PortfolioName;
 use sp_std::{convert::TryFrom, prelude::*};
 
+const PORTFOLIO_NAME_LEN: u32 = 500;
+
 benchmarks! {
     where_clause { where T: TestUtilsFn<AccountIdOf<T>> }
 
     _ {}
 
     create_portfolio {
-        // Length of portfolio name
-        let i in 1 .. 500;
```

Comment:
Since you introduced `PORTFOLIO_NAME_LEN` maybe use it for the bench params also?